lib/Fmt_ast.ml

```diff
@@ -1374,7 +1368,8 @@ and fmt_pattern ?ext c ?pro ?parens ?(box = false)
         | `Fit_or_vertical | `Vertical -> open_hvbox
         | `Fit | `Nested | `Toplevel | `All -> open_hovbox
       in
-      hvbox 0
+      hvbox
+        (if nested then -2 else 0)
```

Comment:
What diff does this cause? I don't see any diff in the test output that I'd expect this to correspond to.